src/matchmaking/match/match.cpp

```diff
@@ -340,7 +339,7 @@ bool Match::adjudicate(Player& us, Player& them) noexcept {
         data_.termination = MatchTermination::ADJUDICATION;
         data_.reason      = us.engine.getConfig().name;
 
-        if (us.engine.lastScore() < tournament_options_.resign.score) {
+        if (us.engine.lastScore() < 0) {
```

Comment:
resign score is now useless?

Reply:
yeah its not needed there, its better this way actually, since there will no longer be cases where the engine with positive mate score get wrongly adjudicated as a loss

Reply:
then we can remove the option?

Reply:
> then we can remove the option?

no i meant its not needed in match.cpp, the one in match.hpp is still necessary

Reply:
ah right thanks